### ISSUE TITLE
Not support big excel file

### ISSUE THREAD

**Issue by zotix** [kz26/PyExcelerate#43]
I try to write data about Row 22000 Col 400 to excel and it told me memory error.

**Comment by kz26** [comment]
Sorry, could you rephrase your statement?

**Comment by dukebody** [comment]
I guess he wanted to say that there is no support for writing very big Excel files.

IMHO if someone is working with big data, she'd better avoid Excel. ;)

**Comment by kevmo314** [comment]
Closing this issue since there haven't been any updates. Unfortunately, we have to store the xlsx file in memory prior to writing it as it has to be compressed, and it sounds like OP is consuming too much memory for the write. Opting to turn off compression to allow directly streaming to disk might be something worth investigating in the future, however I'd be surprised if it were productive since there are a number of other limits you'd run into when trying to write massive files to disk then opening them in Excel. I'll reinvestigate if anyone else has a common use case where they're running into memory issues.
